.github/workflows/build.yml

```diff
@@ -133,7 +133,10 @@ jobs:
           EOF
       - run: cargo make -e BUILDSYS_VARIANT=${{ matrix.variant }} unit-tests
       - run: cargo make -e BUILDSYS_VARIANT=${{ matrix.variant }} check-fmt
-      - run: cargo make -e BUILDSYS_VARIANT=${{ matrix.variant }} check-lints
+      # Only run clippy and shell linters (avoids also running Go lint checkers)
+      - run: |
```

Comment:
Can this be two separate runs instead?

Reply:
`check-shell` is another one that's not variant-dependent, so it would be nice to break that out into a different workflow that only runs once on a PR. But that's good for a follow up.